lets-talk.fifthtry.site/state.ftd

```diff
@@ -111,3 +111,18 @@ leaveMeeting()
 js: $assets.files.js.functions.js
 
 endMeeting()
+
+-- void toggle-mic():
+js: $assets.files.js.functions.js
```

Comment:
Same here, we need explicit versions of these. I am assuming there are variables in state.ftd that tell you the current state of each of these.

Reply:
done in https://github.com/fifthtry-community/lets-talk/pull/8/commits/427c5a1b3cda3d419bd97997932dece8b60b4cd5.
Yes, you can know mic/scree/video using `state.self.{mic, screen, video}`, participants also have this data.